Steps4Impact/Journey/Cells/MilestoneCell.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  CurrentMilestoneCell.swift
+//  Steps4Impact
+//
+//  Created by Aalim Mulji on 11/17/19.
+//  Copyright © 2019 AKDN. All rights reserved.
+//
+
+import UIKit
+
+struct MilestoneContext : CellContext {
+  var identifier: String = MilestoneCell.identifier
+  var sequence = 0
+  var distance = 0
+  var name = ""
+  var flagName = ""
+  var journeyMap = ""
+  var description = ""
+  var title = ""
+  var subTitle = ""
+  var media = ""
+  var content = ""
+  var status : MilestoneStatus = .notCompleted
+}
+
+class MilestoneCell: ConfigurableTableViewCell {
+
+  static let identifier: String = "MilestoneCell"
+
+  let circle : UIView = {
+     var view = UIView(frame: .zero)
+     view.translatesAutoresizingMaskIntoConstraints = false
+     view.layer.cornerRadius = 12
+     return view
+   }()
+
+   let verticalBar : UIView = {
+     var view = UIView(frame: .zero)
+     view.translatesAutoresizingMaskIntoConstraints = false
+     view.frame.size.width = 2
+     return view
+   }()
+
+   let milestoneCountLabel : UILabel = {
+     var label = UILabel()
+     label.translatesAutoresizingMaskIntoConstraints = false
+     label.font = UIFont.systemFont(ofSize: 14, weight: .regular)
+     return label
+   }()
+
+   let milestoneNameButton : UIButton = {
+     var button = UIButton()
+     button.translatesAutoresizingMaskIntoConstraints = false
+    button.contentHorizontalAlignment = .left
+    button.titleLabel?.font = UIFont.systemFont(ofSize: 16, weight: .regular)
+     return button
+   }()
+
+  override func commonInit() {
+    super.commonInit()
+
+    contentView.addSubview(circle) {
+      $0.top.equalToSuperview()
+      $0.leading.equalToSuperview().offset(Style.Padding.p16)
+      $0.width.height.equalTo(Style.Size.s24)
+    }
+    contentView.addSubview(milestoneCountLabel) {
+      $0.top.equalToSuperview()
+      $0.leading.equalTo(circle.snp.trailing).offset(Style.Padding.p12)
+      $0.trailing.equalToSuperview().inset(Style.Padding.p12)
+      $0.height.equalTo(Style.Size.s24)
+    }
+    contentView.addSubview(milestoneNameButton) {
+      $0.top.equalTo(milestoneCountLabel.snp.bottom).offset(Style.Padding.p4)
+      $0.leading.equalTo(circle.snp.trailing).offset(Style.Padding.p12)
+      $0.trailing.equalToSuperview().inset(Style.Padding.p12)
+      $0.height.equalTo(Style.Size.s24)
+      $0.bottom.equalToSuperview().inset(Style.Size.s64)
+    }
+    contentView.addSubview(verticalBar) {
+      $0.top.bottom.equalToSuperview()
+      $0.centerX.equalTo(circle.snp.centerX)
```

Comment:
you don't to reference the specific anchor if they are. the same. You can just write `$0.centerX.equalTo(circle)`